res/controllers/Hercules_DJControl_Jogvision-scripts.js

```diff
@@ -0,0 +1,310 @@
+    engine.setParameter("[EffectRack1_EffectUnit1_Effect3]", "meta", 0.6);
+    engine.setParameter("[EffectRack1_EffectUnit2_Effect1]", "meta", 0.6);
+    engine.setParameter("[EffectRack1_EffectUnit2_Effect2]", "meta", 0.6);
+    engine.setParameter("[EffectRack1_EffectUnit2_Effect3]", "meta", 0.6);
+    engine.setParameter("[EffectRack1_EffectUnit1]", "mix", 0.6);
+    engine.setParameter("[EffectRack1_EffectUnit2]", "mix", 0.6);
+    engine.setParameter("[QuickEffectRack1_[Channel1]]", "super1", 0.5);
+    engine.setParameter("[QuickEffectRack1_[Channel2]]", "super1", 0.5);
+
+    // Connect the VUMeters and Jog Inner LED
+    engine.connectControl("[Channel1]", "VuMeter", "DJCJV.vuMeterUpdateDA");
+    engine.connectControl("[Channel2]", "VuMeter", "DJCJV.vuMeterUpdateDB");
+    engine.connectControl("[Channel1]", "playposition", "DJCJV.wheelInnerUpdateDA");
+    engine.connectControl("[Channel2]", "playposition", "DJCJV.wheelInnerUpdateDB");
+
+    // Enable wheels outer leds rotation by timer (when channel is playing)s
+    DJCJV.ledRotationTimer = engine.beginTimer(20, function() {
+        if (engine.getValue("[Channel1]", "play") == 1) {
+            midi.sendShortMsg(0xB0, 0x60, dec2hex(DJCJV.rotationA));
+            DJCJV.rotationA = DJCJV.rotationA > 127 ? 1 : DJCJV.rotationA + 1;
+        }
+        if (engine.getValue("[Channel2]", "play") == 1) {
+            midi.sendShortMsg(0xB1, 0x60, dec2hex(DJCJV.rotationB));
+            DJCJV.rotationB = DJCJV.rotationB > 127 ? 1 : DJCJV.rotationB + 1;
+        }
+    });
+
+    // Connect the the beat_active with beat leds
+    engine.connectControl("[Channel1]", "beat_active", "DJCJV.beatActiveDA");
+    engine.connectControl("[Channel2]", "beat_active", "DJCJV.beatActiveDB");
+    engine.connectControl("[Channel1]", "stop", "DJCJV.beatActiveClearDA");
+    engine.connectControl("[Channel2]", "stop", "DJCJV.beatActiveClearDB");
+
+    // Ask the controller to send all current knob/slider values over MIDI, which will update the corresponding GUI controls in MIXXX.
+    midi.sendShortMsg(0xB0, 0x7F, on);
+
+};
+
+// Finalization
+DJCJV.shutdown = function() {
+    if (DJCJV.ledRotationTimer) {
+        engine.stopTimer(DJCJV.ledRotationTimer);
+    }
+    midi.sendShortMsg(0xB0, 0x7F, off);
+    midi.sendShortMsg(0xB1, 0x7F, off);
+    midi.sendShortMsg(0x90, 0x7F, off);
+    midi.sendShortMsg(0x91, 0x7F, off);
+};
+
+// Beat led display
+DJCJV.beatActiveDA = function(value, group, control) {
+    if (value == 1) {
+        return;
+    }
+    midi.sendShortMsg(0x90, 0x3A, DJCJV.beatAccumDA == 1 ? on : off);
+    midi.sendShortMsg(0x90, 0x3B, DJCJV.beatAccumDA == 2 ? on : off);
+    midi.sendShortMsg(0x90, 0x3C, DJCJV.beatAccumDA == 3 ? on : off);
+    midi.sendShortMsg(0x90, 0x3D, DJCJV.beatAccumDA == 4 ? on : off);
+    DJCJV.beatAccumDA = DJCJV.beatAccumDA >= 4 ? 1 : DJCJV.beatAccumDA + 1;
+};
+DJCJV.beatActiveDB = function(value, group, control) {

```

Comment:
There's a lot of redundancy here. IMHO the best solution would be to just use a single function and do:
```js
var beatAccum = (group === '[Channel1]' ? DJCJV.beatAccumDA : DJCJV.beatAccumDB);
```